connexion/decorators/validation.py

```diff
@@ -107,12 +107,24 @@ def wrapper(request):
                 data = request.json
 
                 if data is None and len(request.body) > 0 and not self.is_null_value_valid:
-                    # the body has contents that were not parsed as JSON
-                    return problem(415,
-                                   "Unsupported Media Type",
-                                   "Invalid Content-type ({content_type}), expected JSON data".format(
-                                       content_type=request.headers.get("Content-Type", "")
-                                   ))
+                    try:
+                        ctype_is_json = is_json_mimetype(request.headers.get("Content-Type", "") )
+                    except:
```

Comment:
What exceptions are you expecting here, and why?
FYI bare excepts are a [python-anti-pattern](https://docs.quantifiedcode.com/python-anti-patterns/correctness/no_exception_type_specified.html#no-exception-type-s-specified)

Reply:
is_json_mimetype is throwing ValueError when passing empty string (in case Content-Type is not present). I'll change to a specific except clause.